pkg/pillar/cmd/zedagent/reportinfo.go

```diff
@@ -654,14 +658,19 @@ func PublishDeviceInfoToZedCloud(ctx *zedagentContext) {
 	//If there are no failures and defers we'll send this message,
 	//but if there is a queue we'll retry sending the highest priority message.
 	withNetTracing := traceNextInfoReq(ctx)
-	zedcloud.SetDeferred(zedcloudCtx, deviceUUID, buf, size,
-		statusUrl, true, withNetTracing, info.ZInfoTypes_ZiDevice)
-	zedcloud.HandleDeferred(zedcloudCtx, time.Now(), 0, true)
+
+	const ignoreErr = LOCDest
+	ditems := createDeferredItemsByDest(ctx, dest, ignoreErr, deviceUUID, buf,
+		size, true, withNetTracing, info.ZInfoTypes_ZiDevice)
+	zedcloud.AddDeferredList(zedcloudCtx, ditems)
+	zedcloud.HandleDeferred(zedcloudCtx, time.Now(), 0, false)
```

Comment:
Why are you changing the last argument to HandleDeferred from true to false here?